com.unity.render-pipelines.high-definition/Editor/VFXGraph/VFXHDRPSubOutput.cs

```diff
@@ -44,6 +44,14 @@ protected override IEnumerable<string> filteredOutSettings
                     yield return "transparentRenderQueue";
                 else
                     yield return "opaqueRenderQueue";
+
+                if (owner is VFXShaderGraphParticleOutput shaderGraphOutput &&
```

Comment:
Minor :  it could be more appropriate to merge this condition with `!supportsQueueSelection` or even integrate this test in `supportsQueueSelection`, I'm not sure it make sense but it can be more consitent.

Reply:
Good point, missed that case :)